app/femr/data/models/mysql/Patient.java

```diff
@@ -53,6 +53,9 @@
     private List<PatientEncounter> patientEncounters;
     @Column(name = "isDeleted", nullable = true)
     private DateTime isDeleted;
+    @Column(name = "deleted_by_user_id", unique = false, nullable = false) 
```

Comment:
Hi Kevin, I have updated my code, now nullable is true.

Thanks,
Vivek

From: Kevin Zurek [mailto:notifications@github.com]
Sent: Sunday, February 21, 2016 2:06 PM
To: FEMR/femr
Cc: ao3742
Subject: Re: [femr] track user deletes patient (#484)


In app/femr/data/models/mysql/Patient.java<https://github.com/FEMR/femr/pull/484#discussion_r53571243>:

> @@ -53,6 +53,9 @@

>      private List<PatientEncounter> patientEncounters;

>      @Column(name = "isDeleted", nullable = true)

>      private DateTime isDeleted;

> +    @Column(name = "deleted_by_user_id", unique = false, nullable = false)

nullable should be true because the column always has a null value unless a patient has been deleted

—
Reply to this email directly or view it on GitHub<https://github.com/FEMR/femr/pull/484/files#r53571243>.